worker-macros/src/event.rs

```diff
@@ -58,12 +58,12 @@ pub fn expand_macro(attr: TokenStream, item: TokenStream) -> TokenStream {
             // create a new "main" function that takes the worker_sys::Request, and calls the
             // original attributed function, passing in a converted worker::Request
             let wrapper_fn = quote! {
-                pub async fn #wrapper_fn_ident(req: worker_sys::Request, env: ::worker::Env) -> worker_sys::Response {
```

Comment:
Macros should use absolute paths (`::worker`)?

So if somebody defines a module named `worker` Rust still knows to resolve `::worker` to worker crate, not the user defined worker module.

Reply:
I thought about that and wasn't sure, what I should use. I'll change it to `::worker` thinking about it again, it makes more sense.